docs/.vuepress/containers/conditionalBlock/conditionalBlock.js

```diff
@@ -0,0 +1,45 @@
+const { getBuildDocsFramework } = require('../../helpers');
+
+/* eslint-disable jsdoc/require-description-complete-sentence */
+/**
+ * Container used to display/hide blocks of content relevant to specific frameworks.
+ * It relies on providing the framework name(s) as arguments to the container (see 'Usage' below) and setting a
+ * `framework` field in front matter as the framework name, for example:
+ *
+ * ```
+ * ---
+ * framework: vue
```

Comment:
Is it still valid? I don't understand which part of code handle it.

Reply:
You're right, not applicable anymore 👍